deployment/ansible/roles/cac-tripplanner.otp-data/files/validate_feed.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/python
+
+# Check GTFS feeds have no errors and are current, using feedvalidator.py
```

Comment:
It's based on my [validator script for the PATCO GTFS](https://github.com/flibbertigibbet/patco-gtfs/blob/develop/validate_feed.py) that I use with Travis, so yes to both.  It's just a tiny script for a convenience check, so it doesn't seem to me to be something to fill with docstrings (we'll never build out docs for it.)